### ISSUE TITLE
Allow passing project directory to command instead of path to composer.json

### ISSUE THREAD

**Issue by LeoVie** [maglnet/ComposerRequireChecker#139]
Given you have a project in `/var/www/foo-project`.

Then you have to call `composer-require-checker check /var/www/foo-project/composer.json`.

It would be nice if you could just pass the directory to the command: `composer-require-checker check /var/www/foo-project` and the command would check if there is a `composer.json` in this directory.

I guess this could be reached by extending the constructor of `ComposerRequireChecker\JsonLoader` like this:
```php
public function __construct($path)
    {
       if (strpos($path, 'composer.json') === false) {
          $path = $path . DIRECTORY_SEPARATOR . 'composer.json';
        }
        if (!is_readable($path) || ($content = file_get_contents($path)) === false) {
            throw new NotReadableException('unable to read ' . $path);
        }
        $this->data = json_decode($content, true);
        if ($this->data === null && JSON_ERROR_NONE !== json_last_error()) {
            throw new InvalidJsonException('error parsing ' . $path . ': ' . json_last_error_msg());
        }
    }
```

**Comment by LeoVie** [comment]
I guess this could be reached by extending the constructor of `ComposerRequireChecker\JsonLoader` with the following:
```php
if (strpos('composer.json', $path) === false) {
    $path = $path . DIRECTORY_SEPARATOR . 'composer.json';
}
```

**Comment by Ocramius** [comment]
I'd say that running with an explicit `composer.json` is sufficient, while adding code to the repo to support another use-case is more problematic.

Closing as `won't fix` here.
